jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver/jbpm-7.69-to-7.70-TimerMappingInfo.sql

```diff
@@ -1 +1 @@
-ALTER TABLE TimerMappingInfo ADD COLUMN info varbinary(MAX);
```

Comment:
we should also do the same change for the following script:

- https://github.com/kiegroup/jbpm/blob/main/jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver2008/jbpm-7.69-to-7.70-TimerMappingInfo.sql

Also, should we keep this file named as is, or should we create a new one named jbpm-7.70-to-7.71-TimerMappingInfo.sql? wdyt?

Reply:
thanks for pointing out @afalhambra, good catch!
I think we can keep it as it is (even if 7.70 is released) because the feature belongs to that version, and these are static scripts. @elguardian wdyt?